modules/examples/src/main/java/com/ibm/cloud/platform_services/context_based_restrictions/v1/ContextBasedRestrictionsExamples.java

```diff
@@ -0,0 +1,357 @@
+import com.ibm.cloud.sdk.core.service.exception.ServiceResponseException;
+import com.ibm.cloud.sdk.core.util.CredentialUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+
+//
+// This file provides an example of how to use the Context Based Restrictions service.
+//
+// The following configuration properties are assumed to be defined:
+// CONTEXT_BASED_RESTRICTIONS_URL=<service base url>
+// CONTEXT_BASED_RESTRICTIONS_AUTH_TYPE=iam
+// CONTEXT_BASED_RESTRICTIONS_APIKEY=<IAM apikey>
+// CONTEXT_BASED_RESTRICTIONS_AUTH_URL=<IAM token service base URL - omit this if using the production environment>
+// CONTEXT_BASED_RESTRICTIONS_TEST_ACCOUNT_ID=<the id of the account under which test CBR zones and rules are created>
+// CONTEXT_BASED_RESTRICTIONS_TEST_SERVICE_NAME=<the name of the service to be associated with the test CBR rules>
+//
+// These configuration properties can be exported as environment variables, or stored
+// in a configuration file and then:
+// export IBM_CREDENTIALS_FILE=<name of configuration file>
+//
+public class ContextBasedRestrictionsExamples {
+    private static final Logger logger = LoggerFactory.getLogger(ContextBasedRestrictionsExamples.class);
+
+    protected ContextBasedRestrictionsExamples() {
+    }
+
+    @SuppressWarnings("checkstyle:methodlength")
+    public static void main(String[] args) throws Exception {
+        ContextBasedRestrictions contextBasedRestrictionsService = ContextBasedRestrictions.newInstance();
+        String serviceURL = "";
+        String testAccountID = "";
+        String testServiceName = "";
+        String zoneID = "";
+        String zoneRev = "";
+        String ruleID = "";
+        String ruleRev = "";
+        // Load up our test-specific config properties.
+        Map<String, String> config = CredentialUtils.getServiceProperties(ContextBasedRestrictions.DEFAULT_SERVICE_NAME);
+
+        serviceURL = config.get("URL");
+        testAccountID = config.get("TEST_ACCOUNT_ID");
+        testServiceName = config.get("TEST_SERVICE_NAME");
+
+        try {
+            System.out.println("createZone() result:");
+            // begin-create_zone
+            AddressIPAddress addressModel = new AddressIPAddress.Builder()
+                    .type("ipAddress")
+                    .value("169.23.56.234")
+                    .build();
+            CreateZoneOptions createZoneOptions = new CreateZoneOptions.Builder()
+                    .name("SDK TEST - an example of zone")
+                    .accountId(testAccountID)
+                    .description("SDK TEST - this is an example of zone")
+                    .addresses(new java.util.ArrayList<Address>(java.util.Arrays.asList(addressModel)))
+                    .build();
+
+            Response<Zone> response = contextBasedRestrictionsService.createZone(createZoneOptions).execute();
+            Zone zone = response.getResult();
+
+            System.out.println(zone);
```

Comment:
```suggestion
            System.out.println(zone);
            // end-create_zone
```

Reply:
This way we ensure that the 105 and 106 lines will not be displayed in the reference, so our users won't be confused what these lines for.